homeassistant/components/prometheus/__init__.py

```diff
@@ -369,6 +374,38 @@ def _handle_person(self, state):
         value = self.state_as_number(state)
         metric.labels(**self._labels(state)).set(value)
 
+    def _handle_cover(self, state):
+        metric = self._metric(
+            "cover_state",
+            self.prometheus_cli.Gauge,
+            "State of the cover (0/1)",
+            ["state"],
+        )
+
+        cover_states = [STATE_CLOSED, STATE_CLOSING, STATE_OPEN, STATE_OPENING]
+        for cover_state in cover_states:
+            metric.labels(**dict(self._labels(state), state=cover_state)).set(
+                float(cover_state == state.state)
+            )
+
+        position = state.attributes.get(ATTR_POSITION)
+        if position is not None:
+            position_metric = self._metric(
+                "cover_position",
+                self.prometheus_cli.Gauge,
+                "Position of the cover (0/100)",
```

Comment:
This should probably be `0-100` rather than `0/100` (same for cover_tilt_position below).

Reply:
Changed `position` and `tilt_position` descriptions to `0-100` instead of `0/100`, good catch.